### PR TITLE
all: fix lint errors

### PR DESCRIPTION
**Pull request by creamsoup**

### CODE REVIEW

File: examples/src/main/java/io/grpc/examples/hedging/HedgingHelloWorldServer.java [hunk 1]
```diff
@@ -97,9 +96,9 @@ public void sayHello(HelloRequest req, StreamObserver<HelloReply> responseObserv
   static class LatencyInjectionInterceptor implements ServerInterceptor {
 
     @Override
-    public <HelloRequest, HelloReply> Listener<HelloRequest> interceptCall(
```
Comment (dapengzhang0) [inline examples/src/main/java/io/grpc/examples/hedging/HedgingHelloWorldServer.java]: I was suppose to do the following, but forgot to remove `<HelloRequest, HelloReply>`
```
@Override
public Listener<HelloRequest> interceptCall(

Could the above compile?
```

File: core/src/main/java/io/grpc/internal/InsightBuilder.java [hunk 2]
```diff
@@ -25,7 +26,7 @@
  * information.
  */
 public final class InsightBuilder {
-  private final ArrayList<String> buffer = new ArrayList<String>();
+  private final List<String> buffer = new ArrayList<>();
```
Comment (dapengzhang0) [inline core/src/main/java/io/grpc/internal/InsightBuilder.java]: `ArrayList` is fine (just remove `<String>`). See https://github.com/grpc/grpc-java/pull/5892#discussion_r295381892

File: examples/src/main/java/io/grpc/examples/hedging/HedgingHelloWorldServer.java (hunk 1 above)
Comment (creamsoup) [reply examples/src/main/java/io/grpc/examples/hedging/HedgingHelloWorldServer.java]: unfortunately, no.
/usr/local/google/home/jihuncho/git/grpc-java/examples/src/main/java/io/grpc/examples/hedging/HedgingHelloWorldServer.java:96: error: LatencyInjectionInterceptor is not abstract and does not override abstract method <ReqT,RespT>interceptCall(ServerCall<ReqT,RespT>,Metadata,ServerCallHandler<ReqT,RespT>) in ServerInterceptor
  static class LatencyInjectionInterceptor implements ServerInterceptor {
         ^
  where ReqT,RespT are type-variables:
    ReqT extends Object declared in method <ReqT,RespT>interceptCall(ServerCall<ReqT,RespT>,Metadata,ServerCallHandler<ReqT,RespT>)
    RespT extends Object declared in method <ReqT,RespT>interceptCall(ServerCall<ReqT,RespT>,Metadata,ServerCallHandler<ReqT,RespT>)
/usr/local/google/home/jihuncho/git/grpc-java/examples/src/main/java/io/grpc/examples/hedging/HedgingHelloWorldServer.java:99: error: name clash: interceptCall(ServerCall<HelloRequest,HelloReply>,Metadata,ServerCallHandler<HelloRequest,HelloReply>) in LatencyInjectionInterceptor and <ReqT,RespT>interceptCall(ServerCall<ReqT,RespT>,Metadata,ServerCallHandler<ReqT,RespT>) in ServerInterceptor have the same erasure, yet neither overrides the other
    public Listener<HelloRequest> interceptCall(
                                  ^
  where ReqT,RespT are type-variables:
    ReqT extends Object declared in method <ReqT,RespT>interceptCall(ServerCall<ReqT,RespT>,Metadata,ServerCallHandler<ReqT,RespT>)
    RespT extends Object declared in method <ReqT,RespT>interceptCall(ServerCall<ReqT,RespT>,Metadata,ServerCallHandler<ReqT,RespT>)
/usr/local/google/home/jihuncho/git/grpc-java/examples/src/main/java/io/grpc/examples/hedging/HedgingHelloWorldServer.java:98: error: method does not override or implement a method from a supertype
    @Override

File: core/src/main/java/io/grpc/internal/InsightBuilder.java (hunk 2 above)
Comment (creamsoup) [reply core/src/main/java/io/grpc/internal/InsightBuilder.java]: okay.

**Pull request merged**